test/integration/wix/__snapshots__/fetch-from-wix-har.test.js.snap

```diff
@@ -95,6 +95,569 @@ exports[`wix: empty.har 1`] = `
+<p></p>
+<!-- /wp:paragraph -->]]></content:encoded>
+		<wp:comment_status><![CDATA[closed]]></wp:comment_status>
+		<wp:status><![CDATA[publish]]></wp:status>
+		<wp:post_type><![CDATA[page]]></wp:post_type>
+		<wp:is_sticky>0</wp:is_sticky>
+	</item>
+	<item>
+		<wp:post_id>31</wp:post_id>
+		<title><![CDATA[Gallery Page]]></title>
+		<description></description>
+		<content:encoded><![CDATA[<!-- wp:gallery {\\"ids\\":[\\"dataItem-bJGg3Al\\",\\"dataItem-pjYdtZX\\",\\"dataItem-c9EZbPBo\\",\\"dataItem-ix8o1r0W\\"]} -->
+<figure class=\\"wp-block-gallery columns-3 is-cropped\\"><ul class=\\"blocks-gallery-grid\\"><li class=\\"blocks-gallery-item\\"><figure><img src=\\"https://music.wixstatic.com/mp3/v2RHz_ZlMU0yd1JEYcA5J4uvA2zA9rbrjsKoc~mv2_d_3000_2930_s_4_2.jpg\\" alt=\\"\\" data-id=\\"dataItem-bJGg3Al\\" class=\\"wp-image-dataItem-bJGg3Al\\"/></figure></li><li class=\\"blocks-gallery-item\\"><figure><img src=\\"https://music.wixstatic.com/mp3/XZKol_LAhGy0RqaBgitZJJLW6RnMHlSiKXUEn~mv2_d_3000_1946_s_2.jpg\\" alt=\\"\\" data-id=\\"dataItem-pjYdtZX\\" class=\\"wp-image-dataItem-pjYdtZX\\"/></figure></li><li class=\\"blocks-gallery-item\\"><figure><img src=\\"https://music.wixstatic.com/mp3/X84P2_eLeeOFhl0dTWnhLv81eFGp4oulwfpSf~mv2_d_3000_1965_s_2.jpg\\" alt=\\"\\" data-id=\\"dataItem-c9EZbPBo\\" class=\\"wp-image-dataItem-c9EZbPBo\\"/></figure></li><li class=\\"blocks-gallery-item\\"><figure><img src=\\"https://music.wixstatic.com/mp3/i1QDX_dWMHQcXZv2CwUmAgmWWSJ9LOSAovE0b~mv2_d_3000_1995_s_2.jpg\\" alt=\\"\\" data-id=\\"dataItem-ix8o1r0W\\" class=\\"wp-image-dataItem-ix8o1r0W\\"/></figure></li></ul></figure>
+<!-- /wp:gallery -->]]></content:encoded>
+		<wp:comment_status><![CDATA[closed]]></wp:comment_status>
+		<wp:status><![CDATA[publish]]></wp:status>
+		<wp:post_type><![CDATA[page]]></wp:post_type>
+		<wp:is_sticky>0</wp:is_sticky>
+	</item>
+	<item>
+		<wp:post_id>32</wp:post_id>
+		<title><![CDATA[Media Page]]></title>
+		<description></description>
+		<content:encoded><![CDATA[<!-- wp:heading -->
+<h2>Video, Audio, Spotify, Sound Cloud</h2>
+<!-- /wp:heading -->
+
+<!-- wp:embed {\\"url\\":\\"https://dai.ly/k3jznaTlT1RbXjwkuYx\\",\\"providerNameSlug\\":\\"dailymotion\\"} -->
+<figure class=\\"wp-block-embed is-provider-dailymotion wp-block-embed-dailymotion\\"><div class=\\"wp-block-embed__wrapper\\">
+https://dai.ly/k3jznaTlT1RbXjwkuYx
+</div></figure>
+<!-- /wp:embed -->
+
+<!-- wp:audio -->
+<figure class=\\"wp-block-audio\\"><audio controls src=\\"https://music.wixstatic.com/mp3/dmPSv_DPcDH86Ium3LCRu4N4jYKKCQrhgTVsy.mp3\\"></audio></figure>
+<!-- /wp:audio -->
+
+<!-- wp:embed {\\"url\\":\\"spotify:playlist:37i9dQZF1E4lCYovezyEZT\\"} -->
+<figure class=\\"wp-block-embed\\"><div class=\\"wp-block-embed__wrapper\\">
+spotify:playlist:37i9dQZF1E4lCYovezyEZT
+</div></figure>
+<!-- /wp:embed -->
+
+<!-- wp:embed {\\"url\\":\\"\\\\u0022https://soundcloud.com/samiran-saharia/tales-from-the-other-sideoriginal-mix\\\\u0022\\"} -->
```

Comment:
Not sure about the `\\u0022` here, where does it come from? Looks like the quotes around the URL still need to be stripped.

Reply:
Nice catch @akirk!

The data we got from the SoundCloud component has additional quote marks.

Example:
```"param_font_resolveUrl": "\"https://soundcloud.com/samiran-saharia/tales-from-the-other-sideoriginal-mix\"",```

I'll fix it right away.